.ansible-lint

```diff
@@ -1,8 +1,10 @@
 ---
+exclude_paths:
+  - tests/
```

Comment:
Is excludiding "tests" already needed at this point or only after the tox-lsr update? Do you have a list of failures for the tests directory? @liangwen12year can maybe address them.

Reply:
> Is excludiding "tests" already needed at this point or only after the tox-lsr update?

It will be needed after the update to use ansible-core 2.14 and ansible-lint 6.x

> Do you have a list of failures for the tests directory? @liangwen12year can maybe address them.

It would probably be easier for @liangwen12year to run `tox -e ansible-lint` locally and see the list.  It is a pretty big list . . . nothing really complicated to fix, just a lot of tedious work.

Reply:
> > Is excludiding "tests" already needed at this point or only after the tox-lsr update?
> 
> It will be needed after the update to use ansible-core 2.14 and ansible-lint 6.x

What is the timeline for this? Would it be ok to do the change to exclude `tests` together with the update? Or is this update happening only externally in the CI?

Reply:
> > > Is excludiding "tests" already needed at this point or only after the tox-lsr update?
> > 
> > 
> > It will be needed after the update to use ansible-core 2.14 and ansible-lint 6.x
> 
> What is the timeline for this?

QE is blocked - RHEL 8.8/9.2 have switched to ansible-core 2.14 - it isn't just the network role that has issues with ansible-core 2.14, but network QE is ready and waiting

> Would it be ok to do the change to exclude `tests` together with the update? Or is this update happening only externally in the CI?

Not sure what you mean.

Reply:
Thank you, now I understand what is updated. I was assuming it would be some configuration in the role's files or the CI. @liangwen12year please track the need to update the tests so it can be planned for in a sprint.

Reply:
Note that ansible-lint 6.x has an option `--write` - this will attempt to fix some of the issues - but you'll have to read the docs, and configure it carefully - by default it will change things like line wrapping which we do not want to touch.